docs/devGuide/bootcamp/contributeToDocs.md

```diff
@@ -91,6 +91,40 @@ Check that the following are true:
 
 </box>
 
+## Adding Intra-Site Links to Documentation
```

Comment:
Let's shift this into Workflow (create a Docs section in there), as the content included here is phrased in a BootCamp task manner.